src/particles/transformation/CoordinateTransformation.cpp

```diff
@@ -60,9 +57,11 @@ namespace transformation {
                     amrex::ParticleReal const pzd = sqrt(pow(pd, 2) - 1.0);
 
                     ToFixedS const to_s(pzd);
+                    auto tile_data = pti.GetParticleTile().getParticleTileData();
```

Comment:
This probably needs to be a reference.

But this probably does not copy to GPU, does it?
```suggestion
                    auto & tile_data = pti.GetParticleTile().getParticleTileData();
```